base/sparse/sparsematrix.jl

```diff
@@ -2268,22 +2268,25 @@ function spset!{Tv,Ti<:Integer}(A::SparseMatrixCSC{Tv}, x::Tv, I::AbstractVector
     ((I[end] > m) || (J[end] > n)) && throw(DimensionMismatch(""))
     nnzA = nnz(A) + lenI * length(J)
 
-    colptr = A.colptr
     rowvalA = rowval = A.rowval
     nzvalA = nzval = A.nzval
 
     rowidx = 1
     nadd = 0
-    @inbounds for col in 1:n
-        rrange = colptr[col]:(colptr[col+1]-1)
-        (nadd > 0) && (colptr[col] = colptr[col] + nadd)
+    for col in 1:n
```

Comment:
Accidental, thanks for spotting. Removed when testing some things and forgot to add back. @jrevels, What happens if I force push to this branch?